tds-itemselection-impl/src/main/java/tds/itemselection/impl/item/CsetItem.java

```diff
@@ -136,9 +136,15 @@ public CsetGroup getParentGroup () {
   // / Cset1 factory may remove this item from the pool even though the base
   // item is active
   // / </summary>
+  public boolean isActive (boolean ignoreParent)
+  {
+  return !pruned && this.isActive && !ItemUsed &&
+          (ignoreParent ? true : !_parentGroup.getUsed ());
```

Comment:
this can be `(ignoreParent || !_parentGroup.getUsed())` and i think it is cleaner